packages/usa-combo-box/src/combo-box.js

```diff
@@ -210,7 +210,7 @@ const enhanceComboBox = (_comboBoxEl) => {
   const input = document.createElement("input");
   input.setAttribute("id", selectId);
   input.setAttribute("aria-owns", listId);
-  input.setAttribute("aria-autocomplete", "list");
```

Comment:
@mejiaj Wanted to confirm that you intended to remove this. I see this line still present in both [develop](https://github.com/uswds/uswds/blob/develop/src/js/components/combo-box.js#L214) and [library--develop](https://github.com/uswds/uswds/blob/library--develop/packages/usa-combo-box/src/combo-box.js#L213)

Reply:
Good catch, no that wasn't intentional. Fixed in 8c753da0529df4056f011805033f584286177ebf.